pkgs/applications/office/vnote/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, fetchFromGitHub, qt5, hicolor-icon-theme, makeDesktopItem }:
+
+let
+  description = "A note-taking application that knows programmers and Markdown better";
+  desktopItem = makeDesktopItem {
+    name = "VNote";
+    exec = "vnote";
+    icon = "vnote";
+    comment = description;
+    desktopName = "VNote";
+    categories = "Office";
+  };
+in stdenv.mkDerivation rec {
+  version = "2.5";
+  name = "vnote-v${version}";
+
+  src = fetchFromGitHub {
+    owner = "tamlok";
+    repo = "vnote";
+    fetchSubmodules = true;
+    rev = "8d780bd5898464139e252623ffaccd966e093ff1";
```

Comment:
rev = "v${version}";

Reply:
It pulls sources and not artifacts by tag. That is why it is directly referencing specific revision associated with the release.

Reply:
AFAIK `fetchFromGitHub` grabs the source zip, even when a tag is specified. (not all projects have binary releases anyway, despite having release tags)

Reply:
fixed, will push shortly